### ISSUE TITLE
[Extreme Lag] Optifine 1.16.2 + 3

### ISSUE THREAD

**Issue by EtornaZ** [sp614x/optifine#4796]
This started the day 1.16.3 released. I logged into a realm on the official 1.16.2 OptiFine version, and was met with 1 fps every 4-6 seconds. I tried playing a single-player world, and the exact same issue occurred. I went back to a beta OptiFine version for 1.16.2 to try that as a workaround, but the same thing happened. I can't even change things in the game menu such as video settings and game volume without the lag. 

Reproducing the bug is easy as it still happens to me with OptiFine every time I try logging into a world. I was using 1.16.2, but also tried it with the latest 1.16.3 OptiFine betas, and the same thing happens.

I installed OptiFine through the standard mirror download link on the official OptiFine website. There are no crash files, and if a log file is needed, I will gladly give one.

Here is my F3 menu on a singleplayer world.

![2020-09-11_18 23 25](https://user-images.githubusercontent.com/33297131/92977846-14b81b80-f45c-11ea-958f-71f924c9d43a.png)

This does not happen in vanilla Minecraft, only OptiFine.
Please let me know if I need to give you any more information. Thank you!

**Comment by Frakcool** [comment]
I was getting this issue, not as drastically as 1 FPS on 1.16.1 and 1.16.3 versions, for 1.16.2 it was the smoothest gameplay I've ever had, even on jungle biome I was getting constant 60 FPS (capped at that), playing on Single Player world.

Please let me know what kind of extra details are needed and how can I provide them

**Comment by KaiAF** [comment]
Hello, are you still having this issue?

**Comment by EtornaZ** [comment]
Yes, I am. It's for the latest official version (1.16.5), and I thought I had found a way around it that partially works. There was a chance the issue would go away when I restarted my computer, however it usually took several attempts and then after a few sessions of booting Optifine up with it working normally, the issue would come back. I've now stopped using Optifine entirely because of the issue, but would love to be able to go back to it again.
